tests/encrypt/enc-test.sh

```diff
@@ -6,18 +6,16 @@ if [ ! -d ./certs/ ]; then
 fi
 
 run() {
-    RESULT=`./wolfssl $1`
+    RESULT=`eval $1`
```

Comment:
This script won't run on my Mac. The OS thinks it is malicious!
<img width="335" alt="Screen Shot 2021-12-29 at 11 14 58 AM" src="https://user-images.githubusercontent.com/2692870/147696067-c035fbc7-2069-40b8-83a4-6f1a6bec2cee.png">
I have NOT found a way to override this. I am on macOS 11.6.1 (20G224)
But all looks fine in this PR otherwise.